sqlparser/seata-sqlparser-druid/src/main/java/io/seata/sqlparser/druid/DruidIsolationClassLoader.java

```diff
@@ -45,6 +50,28 @@
         return super.loadClass(name, resolve);
     }
 
+    /**
+     * While loading resource files, the resources under the current layer classLoader's path and the parent
+     * classLoader's path will be loaded at the same time.{@link ClassLoader#getResources(String)}
+     * So we rewrite this method to avoid repeated loading of druid resources which in the
+     * {@link #DRUID_RESOURCE_PREFIX}
+     */
+    @Override
+    public Enumeration<URL> getResources(String name) throws IOException {
+        for (String prefix : DRUID_RESOURCE_PREFIX) {
+            if (name.startsWith(prefix)) {
+                return loadInternalResource(name);
+            }
+        }
+        return super.getResources(name);
+    }
+
+    private Enumeration<URL> loadInternalResource(String name) throws IOException {
+        Enumeration<URL>[] tmp = (Enumeration<URL>[])new Enumeration<?>[1];
+        tmp[0] = findResources(name);
+        return new CompoundEnumeration<>(tmp);
```

Comment:
Only one Enumeration, is it necessary to use CompoundEnumeration even loadInternalResources itself?